app/controllers/EventSelectionController.scala

```diff
@@ -71,9 +84,25 @@ class EventSelectionController @Inject()(val controllerComponents: MessagesContr
                     reportVersion = answers.eventDataIdentifier(eventType).version))
                 Redirect(EventSelectionPage.navigate(waypoints, answers, answers).route)
               }
-            case _ => Future.successful(Ok(view(form, waypoints)))
+            case _ =>
+              eventReportingConnector.getFeatureToggle(ltaAbolitionShowHideToggle).flatMap { toggleData =>
+                val displayEventList: Seq[RadioItem] = getFilteredOptions(toggleData.isEnabled, request.userAnswers)
+                Future.successful(Ok(view(form, displayEventList, waypoints)))
+              }
           }
         }
       )
   }
+
+  private def getFilteredOptions(isEnabled: Boolean, ua: UserAnswers)(implicit messages: Messages): Seq[RadioItem] = {
+    val eventsToRemove: Seq[EventSelection] = TaxYear.getSelectedTaxYear(ua).startYear >= "2024" match {
+      case true => Seq(Event2, Event6, Event7, Event8, Event8A)
+      case false => Seq() // TODO: Include new events once developed Seq(Event25, Event26)
```

Comment:
Event26 is now not going ahead so can amend comment

Reply:
done